javascript/apps/taiga/src/app/shared/attachments/components/attachment/attachment.component.html

```diff
@@ -53,6 +54,7 @@
         </tg-ui-dtable-cell>
         <tg-ui-dtable-cell class="actions-column">
           <a
+            #download
```

Comment:
I don't see this notification implemented:
![image](https://github.com/taigaio/taiga/assets/5970959/dc4ff82f-0819-4680-a26e-d71993cb14ce)